### DISCUSSION TITLE
Allow Git Sync to target local filesystem

### DISCUSSION THREAD

**Discussion by rbedger** [Kong/insomnia#3684] [Ideas]
**Is your feature request related to a problem? Please describe.**
The Git Sync feature doesn't work with my company's self-hosted BitBucket instance, which uses a self-signed, internal certificate.

**Describe the solution you'd like**
Allow the Git Sync feature to simply target a repository on disk, leaving the git push up to the user.

**Describe alternatives you've considered**
n/a

**Additional context**
n/a

**Comment by sethgw** [comment]
☝️

**Comment by wdawson** [comment]
Moving some discussion here from #3768 

Hey folks! An update for this: we're tentatively planning to revisit git sync early next year, the goal being to smooth out the flow in general and we'll be exploring how to have Insomnia act more like a text editor for use-cases like this. Of course, no plan survives contact with the enemy but we know this is one of the big areas to improve on.
